data/courses.yaml

```
@@ -230,3 +230,18 @@
     The students are willing and able to learn Lean. Including Lean showcases logic as a vibrant rather than dormant area of study.
     As projects in 2022 students created formalizations of exercise solutions for my book "Automatic complexity",
     some of which are included at https://github.com/bjoernkjoshanssen/diophantine-lemma
+- name: Programování a dokazování v Leanu
+  instructor: Martin Dvorak
+  location: online
+  website: https://mam.mff.cuni.cz/
+  repo: https://github.com/madvorak/lean-mam
+  tags: ['functional programming', 'intro to proofs']
+  summary: >
+    We are a (korespondenční seminář)[https://cs.wikipedia.org/wiki/Koresponden%C4%8Dn%C3%AD_semin%C3%A1%C5%99] (lit. "correspondence seminar" in the sense of
```

This markdown isn't quite right:
```suggestion
    We are a [korespondenční seminář](https://cs.wikipedia.org/wiki/Koresponden%C4%8Dn%C3%AD_semin%C3%A1%C5%99) (lit. "correspondence seminar" in the sense of
```

---

Thanks for noticing!